### PR TITLE
Change BYOVPC and MachinePool

### PR DESCRIPTION
**Pull request by nirarg**

### CODE REVIEW

File: examples/rosa-classic-with-unmanaged-oidc/main.tf [hunk 1]
```diff
@@ -89,15 +90,15 @@ module "rosa_cluster_classic" {
 # machine-pool
 ############################
 module "rosa_machine_pool" {
-  source = "../../modules/machin-pool"
+  source = "../../modules/machine-pool"
 
-  cluster_id = module.rosa_cluster_classic.cluster_id
-  name = var.machine_pool_name
+  cluster_id   = module.rosa_cluster_classic.cluster_id
+  name         = "${var.cluster_name}-MachinePool"
```
Comment (bardielle) [inline examples/rosa-classic-with-unmanaged-oidc/main.tf:96]: please use lower case 
```suggestion
  name         = "${var.cluster_name}-machine-pool"
```

File: examples/rosa-classic-with-unmanaged-oidc/main.tf [hunk 2]
```diff
@@ -89,15 +90,15 @@ module "rosa_cluster_classic" {
...
   machine_type = var.machine_type
 
   // Should set one of replicas autoscaling_enabled
-  replicas = var.replicas
+  replicas            = var.replicas
```
Comment (bardielle) [inline examples/rosa-classic-with-unmanaged-oidc/main.tf:100]: I thought we said that here should be a default hardcoded replicas 
```suggestion
  replicas            = 3
```

File: examples/rosa-classic-with-unmanaged-oidc/main.tf (hunk 1 above)
Comment (nirarg) [reply examples/rosa-classic-with-unmanaged-oidc/main.tf]: Fixed

File: examples/rosa-classic-with-unmanaged-oidc/main.tf (hunk 2 above)
Comment (nirarg) [reply examples/rosa-classic-with-unmanaged-oidc/main.tf:100]: Added the default value to variables

**Review by bardielle — APPROVED** [review]
/lgtm
/approve

**Pull request merged**